sessionctx/variable/sysvar.go

```diff
@@ -378,7 +378,7 @@ var defaultSysVars = []*SysVar{
 	}, GetGlobal: func(s *SessionVars) (string, error) {
 		return strconv.FormatUint(uint64(atomic.LoadUint32(&DDLSlowOprThreshold)), 10), nil
 	}},
-	{Scope: ScopeInstance, Name: TiDBForcePriority, skipInit: true, Value: mysql.Priority2Str[DefTiDBForcePriority], SetGlobal: func(s *SessionVars, val string) error {
```

Comment:
SkipInit has no effect here, because there is no session scope: https://github.com/pingcap/tidb/blob/933d5523cbb9dacc59ce975a2a22de6fd57a9c07/sessionctx/variable/variable.go#L525

The skipInit feature is left over from a refactor PR.